src/Linux/Host/Broker/Impl/Services/LinuxSystemInfoService.cs

```diff
@@ -58,10 +73,8 @@ public class LinuxSystemInfoService : ISystemInfoService {
                 foreach (string line in data) {
                     string[] split = line.Split(_memInfoSplitter, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Can't values have spaces in them (not numbers, obviously, but in general?). It feels like this should really just split by `:`, and use the `String.Split` overload that specifies the maximum number of subitems (so it only splits on the first colon, not any subsequent ones). Then `Trim` spaces on the values, and parse them if they're numbers.